infra/label_evaluator_step_functions/TRACE_SPEC.md

```
@@ -0,0 +1,531 @@
+  "incomplete_traces": []
+}
+```
+
+---
+
+## Phase 2: Fix Receipt Traces (Per-Receipt Parent)
+
+### 2.1 Goal
+Each receipt gets ONE parent trace (`ReceiptEvaluation`) with ALL child steps nested underneath:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)